frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -266,7 +272,11 @@ export function SavedInsightsEmptyState(): JSX.Element {
                             : title.replace('$CONDITION', `matching these filters`)
                         : title.replace('$CONDITION', 'for this project')}
                 </h2>
-                <p className="empty-state__description">{description}</p>
+                {usingFilters ? (
+                    <p className="empty-state__description">{descriptionOnEmptyFilter}</p>
```

Comment:
Hey @naquiuddin! Thanks for this contribution. Only one note: Given `descriptionOnEmptyFilter` is the same for all tabs, could we just inline it?

Reply:
My Pleasure!

Makes sense. I would suggest we do same for the `description` field as well, which also has same text for all tabs. Please see my latest commit and let me know whether that makes sense.

Reply:
Ah, so the descriptions are indeed all _almost_ the same, but "almost" is the key word – the one for the "Favorited" tabs has "Once you _favorite_ an insight" instead of "_create_", so we still need that differentiated

Reply:
Oh Yeah 😄, I didn't notice that. Thanks,
Reverted that and pushed it again.